src/tracing/trace.test.ts

```diff
@@ -38,12 +38,11 @@ describe('trace', () => {
   })
 
   it('records transaction', async () => {
-    const metadata = { data: { a: 'a', b: 2 }, tags: { is_widget: true } }
+    const metadata = { data: { a: 'a', b: 2 } }
```

Comment:
This test just verified that tags worked - can you change it from `is_widget` to another value?

Reply:
You'll have to update the type of `TraceTags` locally to get this to work :(